tests/e2e/scale/noobaa/test_scale_endpoint.py

```diff
@@ -71,7 +82,10 @@ def _assert_endpoint_count(self, desired_count):
         )
 
     def test_scale_endpoint_and_respin_ceph_pods(
-        self, mcg_job_factory, resource_to_delete
+        self,
```

Comment:
can we remove the `MAX_ENDPOINT_COUNT = 2` as use it as a parameter instead?

Reply:
This parameter is using to determine max replicas HPA deployment.  It can't be removed.